cilium/cmd/bpf_egress_list.go

```diff
@@ -38,7 +38,8 @@ var bpfEgressListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf egress list")
 
-		if err := egressmap.OpenEgressMaps(); err != nil {
+		policyMap, err := egressmap.OpenPinnedPolicyMap()
```

Comment:
Rather than call a function to return the singleton provided by the Hive cell, is there not some way to use the new Cell to provide this Map as a dependency?

Reply:
Maybe, but I'm not sure how to tie hive into this command. As I understand it, we currently do `agent cobra command -> create hive -> execute agent`. This is in a separate cobra command. Also note that here we don't want to create the map, we only want to open it if it already exists. The Cell will actually create the map as well.